languages/id.yml

```diff
@@ -0,0 +1,86 @@
+# format tanggal yang ditampilkan didalam blog
+date_format: "MMM DD, YYYY"
+
+global:
+    home: "Depan"
```

Comment:
would `laman` be more appropriate? what's the difference between these 2 words?

Reply:
`laman` derived from the Malay language, mean home page (id= halaman muka, halaman depan). But nowadays, `laman` is often used in place of `halaman` (page). Because word of `halaman depan` too long, i using `depan` (front)